specification/metadata/providerprefix.md

```diff
@@ -0,0 +1,17 @@
+# Provider Prefix
+
+Used as the prefix for the name of custom columns, useful for the consumer of the data to quickly identify custom column extensions to the FOCUS schema implemented by the provider.
```

Comment:
Do we need two different things here OR would it be simpler for practitioners if providers just used one value for both? One downside I can think of is if there's more than one value for tag prefixes.

@rileyjenk @mike-finopsorg

Reply:
https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/360/files#r1560050489

Reply:
I thought about this then I was worried the value that a provider has used in tags (e.g. `AWS:`) isn't 1:1 to a provider's chosen column prefix (e.g. `aws_`). I guess we can discuss this further when we close out on if we are going to stick with x_ or move towards $ProviderPrefix$_. I am guessing this is possibly a significant conversation to reopen at this stage in the game and it might resolve this indirectly.